### ISSUE TITLE
Error in manpage fido_bio_template(3)

### ISSUE THREAD

**Issue by nbraud** [Yubico/libfido2#120]
`man` complains when rendering `fido_bio_template(3)`:

```
$ man --warnings -l -Z man/fido_bio_template.3 > /dev/null
troff: <standard input>:43: warning [p 1, 7.2i]: can't break line
```

You might want to run this sort of test in CI, to ensure manpage issues do not reoccur.

The issue was found by Lintian when packaging libfido2 (v1.3.0) for Debian.

**Comment by martelletto** [comment]
Hi,

Thank you!

To reproduce the problem, I had to set `MANWIDTH` to 80. From a quick look, this seems to match what Lintian is doing. That said, I have no immediate idea how to break that line, and will need to look further into it.

-p.

**Comment by martelletto** [comment]
Fixed in cff11d0. Thanks!
